src/main/java/slimeknights/tconstruct/library/smeltery/OreCastingRecipe.java

```diff
@@ -4,42 +4,56 @@
 import net.minecraft.item.ItemStack;
 import net.minecraftforge.fluids.Fluid;
 import net.minecraftforge.fluids.FluidStack;
-import net.minecraftforge.oredict.OreDictionary;
-
 import java.util.List;
 
 import slimeknights.mantle.util.RecipeMatch;
+import slimeknights.tconstruct.library.utils.RecipeUtil;
 
 /**
  * A casting recipe that takes its output from an oredict entry.
  * Used for ingot casting etc.
  */
 public class OreCastingRecipe extends CastingRecipe {
 
+  /** @Deprecated use oreName instead */
+  @Deprecated
```

Comment:
I wonder if it wouldn't be better if we made a new class for this, and deprecated the old class. Seems a bit cleaner, but less compatible (although I hope nobody casts to that class without checking for instance first..)

Reply:
I considered that, though I was not sure what to name it.

Reply:
OreCastingRecipe2, OreCastingRecipeNew, BetterOreCastingRecipe, UseThisInsteadOfTheOldOneOreCastingRecipe, the terrible possibilities are endless!
I guess yours or maybe SmartOreCastingRecipe or possibly even just OreDictCastingRecipe.. hrm.